packages/docs/pages/reference/elements/doublesharp.mdx

```diff
@@ -8,12 +8,23 @@ import {
 
 # `<doublesharp>`
 
-Short description of what the `<doublesharp>` tag does.
+The self-closing `<doubleflat/>` tag displays the doubleflat symbol in music notation.
 
 ## Syntax
 
 <SyntaxDisplay name="doublesharp" />
 
 ## Examples
 
-    
\ No newline at end of file
+```ptx-example
+<FRAGMENT parents="p">
+    
+    In the context of music, the doublesharp symbol (<doublesharp/>) can be used to modify a 
+    single note.  It raises the pitch of the note by two semitones. Thus A<doublesharp/> has 
+    the same pitch as a B. Often when a key signature already specifies a note be sharped, 
+    the doublesharp simply indicates that it should be sharped beyond what is specified by 
+    the key signature.
```

Comment:
This seems like too much for an example. Maybe just do the first sentence.
`In the context of music, the doublesharp symbol (<doublesharp/>) can be used to modify a single note.`

Also:
 - please wrap at no more than 80 characters.
 - Remove blank lines from the start and end of the fragment.